lib/boombox/transcoders/video.ex

```diff
@@ -0,0 +1,161 @@
+defmodule Boombox.Transcoders.Video do
```

Comment:
WDYT about merging this with the Audio transcoder bin?

Reply:
Now the way how each of transcoder is built is slightly different, I don't see much repeated code there